### PR TITLE
Add badge to link to Galaxy deployment

### PR DESCRIPTION
**Pull request by sbesson**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # Ansible Role: HAProxy
 
 [![Build Status](https://travis-ci.org/geerlingguy/ansible-role-haproxy.svg?branch=master)](https://travis-ci.org/geerlingguy/ansible-role-haproxy)
```
Comment (manics) [inline README.md]: Would you mind fixing the travis link whilst you're at it?

Comment (joshmoore) [reply README.md]: Done

Comment (sbesson) [reply README.md]: 4d33f32 - I used my Josh bot :)

**Comment by sbesson** [comment]
:+1: I think that's all good for `2.1.1` (we need to skip `2.1.0` as per the semver vs PEP440 incompatibility). My understanding of the state of https://github.com/openmicroscopy/ansible-role-haproxy/pull/1 is that it will require more time and I would not block the release on it.

**Comment by joshmoore** [comment]
Cool. Thanks guys. I'm going to push 2.1.0 _as well_ since that's what I did on the other releases. We can choose to stop doing that in the future (though perhaps soon not necessary as we use `dev1` rather than `m1`)

**Pull request merged**